cli.sh

```diff
@@ -1562,47 +1598,34 @@ is_git_repo() {
 }
 
 #####################################################
-# Updates the update clone in the install folder
-#
-# Sets the ${INSTALL_SCRIPT} variable
+# Safely execute a git command in the standard
+#   clone dir `$1`.
 #
+# Returns: Error code from `git`
+#####################################################
+execute_git() {
+    REPO="$1"
+    shift
+
+    git -C "$REPO" \
+        --work-tree="$REPO" \
+        --git-dir="$REPO/.git" \
+        -c core.hooksPath=/dev/null \
+        "$@"
+}
+
+#####################################################
+#  Creates the release clone if needed.
 # Returns:
 #   1 if failed, 0 otherwise
 #####################################################
-update_release_clone() {
+assert_release_clone() {
```

Comment:
I'd leave this as `update` rather than `assert`, because it doesn't only assert but can also clone as a side effect.

Reply:
the side effect of cloning was really meant to be the assert. Its ok to clone, but its not ok to do an update on the clone if there would be... so I leave this